package.json

```diff
@@ -2,13 +2,13 @@
   "private": true,
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
-    "build": "pulp build --censor-lib --strict",
-    "example": "pulp run -I example --main Example.Main"
+    "build": "pulp build -- --censor-lib --strict",
+    "test": "pulp test"
```

Comment:
Will need to update the Travis yaml to run this rather than example now, but 👍  apart from that.